src/screens/OtherProfile/index.js

```diff
@@ -641,12 +733,10 @@ const styles = StyleSheet.create({
     marginLeft: 18
   },
   profileImage: {
-    width: 94,
-    height: 94,
+    width: 100,
+    height: 100,
     borderRadius: 100,
-    // marginLeft: 3,
-    marginTop: 3,
-    marginBottom: 3
+    marginRight: 22
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The profile image size has been changed. Ensure that this change does not negatively impact the layout or design of the component.
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->